### PR TITLE
remove oddity requirement for nanoforge.

### PR DESCRIPTION
**Pull request by pexel5**
## About The Pull Request

Removed the oddity requirement for printing things.

## Why It's Good For The Game

Fortune asked me.

## Changelog
:cl:
tweak: remove oddity requirement for nanoforge.
/:cl:

### CODE REVIEW

**Comment by SirRichardFrancis** [comment]
Firstly there is absolutely no reason for this to be made, secondly it will screw up loot and make all these items more common, thirdly, one and possibly more items price changes are direct violation to contributing guidelines, you should try reading them.
https://wiki.cev-eris.com/Content_GuidelinesEn

**Comment by FortuneMaster** [comment]
I could't understand a single sentence and how they are related to forge

> Firstly there is absolutely no reason for this to be made, secondly it will screw up loot and make all these items more common, thirdly, one and possibly more items price changes are direct violation to contributing guidelines, you should try reading them.

https://wiki.cev-eris.com/Content_GuidelinesEn

**Comment by SirRichardFrancis** [comment]
> I could't understand a single sentence and how they are related to forge

Cause you're slow. ¯\\\_(ツ)_/¯

**Comment by FortuneMaster** [comment]
> > I could't understand a single sentence and how they are related to forge
> 
> Cause you're slow. ¯\_(ツ)_/¯

or maybe someone answered a wrong PR so everything said ended up being unrelated nonsense 😁

**Comment by SirRichardFrancis** [comment]
> or maybe someone answered a wrong PR so everything said ended up being unrelated nonsense 😁

slow

**Comment by FortuneMaster** [comment]
> slow

well, knowing how to attach images is great👍

File: code/game/machinery/autolathe/autolathe.dm [hunk 1]
```diff
@@ -728,8 +728,8 @@
 
 /obj/machinery/autolathe/proc/can_print(datum/computer_file/binary/design/design_file)
 
-	if(use_oddities && !oddity)
-		return ERR_NOODDITY
+//	if(use_oddities && !oddity)
+//		return ERR_NOODDITY
```
Comment (hyperioo) [inline code/game/machinery/autolathe/autolathe.dm:732]:
```suggestion
```

File: code/game/machinery/autolathe/nanoforge.dm [hunk 2]
```diff
@@ -42,9 +42,9 @@
 	return
 
 /obj/machinery/autolathe/nanoforge/proc/compress_matter(mob/user)
-	if(!inspiration || !inspiration.perk)
-		to_chat(user, SPAN_WARNING("Catalyst not found."))
-		return
+//	if(!inspiration || !inspiration.perk)
+//		to_chat(user, SPAN_WARNING("Catalyst not found."))
+//		return
```
Comment (hyperioo) [inline code/game/machinery/autolathe/nanoforge.dm:47]:
```suggestion
```

File: code/game/machinery/autolathe/nanoforge.dm [hunk 3]
```diff
@@ -92,7 +92,8 @@
 
 /obj/machinery/autolathe/nanoforge/icon_off()
 	. = ..()
-	if(. || !inspiration)
+//	if(. || !inspiration)
```
Comment (hyperioo) [inline code/game/machinery/autolathe/nanoforge.dm:95]:
```suggestion
```

**Review by hyperioo — APPROVED** [review]
No need to let them commented, we can always find them back with the git history if needed.

**Pull request merged**